compiler/aarch64/codegen/OMRInstOpCode.enum

```diff
@@ -705,6 +705,18 @@
 		fmaxd,                                                  	/* 0x1E604800	FMAX      	 */
 		fmins,                                                  	/* 0x1E205800	FMIN      	 */
 		fmind,                                                  	/* 0x1E605800	FMIN      	 */
+	/* Vector Immediate */
+		vmovi16b,                                               	/* 0x4F00E400,	MOVI      	 */
+		vmovi8h,                                                	/* 0x4F008400,	MOVI      	 */
+		vmovi4s,                                                	/* 0x4F000400,	MOVI      	 */
+		vmovi4s_one,                                            	/* 0x4F00C400,	MOVI      	 */
+		movid,                                                  	/* 0x2F00E400,	MOVI      	 */
```

Comment:
There is already `movi0d` defined in the `Move Immediate` section above, with the same encoding.
It is used in `dconstEvaluator()` only for loading +0.0 to a FPR.  Please consider replacing it by `movid`.

Reply:
See Issue #5415 and PR #5421 for this.

Reply:
I added `vmovi2s` for float and removed `movi0d` and `movi0s`.
I also added binary encoding unit test cases for `vmovi2s` and `movid` instructions.